Documentation/guides/building-apps/build-properties.md

```diff
@@ -1472,6 +1472,19 @@ string.
 
 Added in Xamarin.Android 7.2.
 
+## ApplicationIcon
+
+An MSBuild property that can be used as an alternative to
+`/application/@android:icon` in the [`AndroidManifest.xml`](~/android/platform/android-manifest.md)
+file. To opt into this feature you must also enable
+`<GenerateApplicationManifest>true</GenerateApplicationManifest>`.
+This will be the default going forward in .NET 6.
+
+See the [Android documentation][application-element] for further details
+about the requirements for `/manifest/application/@android:icon`.
+
+Support for this property was added in Xamarin.Android 12.2 and .NET 7.
```

Comment:
1. Is Classic getting support for this property?
2. If it is, it would be added in 13.2…which may or may not exist?

Probably better to just omit Classic entirely at this point.